crates/bevy_render/src/primitives/mod.rs

```diff
@@ -95,6 +95,24 @@ impl Aabb {
     pub fn max(&self) -> Vec3A {
         self.center + self.half_extents
     }
+
+    /// Check if the point is at the front side of the plane.
+    /// Referenced from: [AABB Plane intersection](https://gdbooks.gitbooks.io/3dcollisions/content/Chapter2/static_aabb_plane.html)
+    #[inline]
+    pub fn is_forward_plane(
+        &self,
+        p_normal: &Vec3A,
+        p_distance: f32,
```

Comment:
We could perhaps merge these two inputs into a single: `plane: HalfSpace`, this would better align with the use later for Frustum intersections, also the Vec3A already has a size of 4 f32's.